.circleci/config.yml

```diff
@@ -547,6 +547,28 @@ jobs:
       - run:
           name: Install Desktop Dependencies
           command: yarn run build-desktop:install-app-deps
+      - run:
+          name: Build Source
+          command: |
+            make -f desktop/Makefile build-main
+      - when:
+          condition: << pipeline.git.tag >>
+          steps:
+            - run:
+                name: Build Appx
+                command: |
+                  # Note: Windows Store build should be unsigned
+                  make -f desktop/Makefile package ELECTRON_BUILDER_ARGS='--config="electron-builder-appx.json"'
+            - run:
+                name: Clean Up Appx
+                command: |
+                  # FIXME: For some reason this the additional artifacts
```

Comment:
Update: it seems the additional artifacts are being generated by the NSIS portion.